src/java.base/share/classes/jdk/internal/foreign/abi/ppc64/aix/AixPPC64Linker.java

```diff
@@ -48,6 +52,9 @@
  */
 public final class AixPPC64Linker extends AbstractLinker {
 
+    static final Map<String, MemoryLayout> CANONICAL_LAYOUTS =
+            SharedUtils.canonicalLayouts(ValueLayout.JAVA_LONG, ValueLayout.JAVA_LONG, ValueLayout.JAVA_INT);
```

Comment:
This field should be `private` and the entry for `double` needs a different alignment:
```java
    private static final Map<String, MemoryLayout> CANONICAL_LAYOUTS;

    static {
        Map<String, MemoryLayout> layouts = new HashMap(
                SharedUtils.canonicalLayouts(
                        ValueLayout.JAVA_LONG,
                        ValueLayout.JAVA_LONG,
                        ValueLayout.JAVA_INT));
        layouts.put("double", ValueLayout.JAVA_DOUBLE.withByteAlignment(4));
        CANONICAL_LAYOUTS = Map.copyOf(layouts);
	}
```

Reply:
Changed as per the suggestion.